ads-with-uniregistry.rst

```diff
@@ -0,0 +1,73 @@
+.. post:: 
+   :tags: advertising, sustainability, draft
+
+Uniregistry sponsors Read the Docs and Open Source
+==================================================
+
+Today we're excited to announce an important sponsorship partner in our `Ethical Advertising`_ campaign: `Uniregistry`_.  Our goal with our ethical advertising program is to provide important funding for open source, and show that it can be done ethically.
+
+Domain registration was identified early as a natural partner to our program, because it sits in the stack of necessary infrastructure for all of us that work on making the internet. We wanted the right partner, because historically we feel that domain registration companies left room for improvement in the customer experience. This is post covers a few of the criteria we used to reach the conclusion to partner with Uniregistry.
+
+To not bury the lede,
+we're offering 25% off first year registrations and/or transfers into Uniregistry,
+using the code ``DOCS25``.
+We hope that you'll enjoy getting started with the service,
+and find it as useful.
+`Sign up`_ now to get started.
+
+Here are some of the reasons that Uniregistry is a good domain host:
```

Comment:
Perhaps something less blunt here. Also, can combine this with the paragraph above if we aren't ready to make an important point with this paragraph. Hastily written example of reduced bluntness: "Purchasing domain names with Uniregistry is easy and secure, and you'll find a number of features that you won't find with other registrars:"